modules/checking-mco-status.adoc

```diff
@@ -14,35 +14,53 @@ To see the status of the Machine Config Operator, its sub-components, and the re
 [source,terminal]
 ----
 $ oc get machineconfigpool
-NAME      CONFIG                  UPDATED  UPDATING   DEGRADED  MACHINECOUNT  READYMACHINECOUNT  UPDATEDMACHINECOUNT DEGRADEDMACHINECOUNT  AGE
-master    rendered-master-dd…     True     False      False     3             3                  3                                0                     4h42m
-worker    rendered-worker-fde…    True     False      False     3             3                  3                                0                     4h42m
 ----
 +
-In the previous output, there are three master and three worker nodes. All machines are updated and none are currently updating. Because all nodes are `Updated` and `Ready` and none are `Degraded`, you can tell that there are no issues.
-
-. To see each existing `machineconfig`, type:
-+
 [source,terminal]
 ----
-$ oc get machineconfigs
-NAME                             GENERATEDBYCONTROLLER          IGNITIONVERSION  AGE
-00-master                        2c9371fbb673b97a6fe8b1c52...   3.2.0            5h18m
-00-worker                        2c9371fbb673b97a6fe8b1c52...   3.2.0            5h18m
-01-master-container-runtime      2c9371fbb673b97a6fe8b1c52...   3.2.0            5h18m
-01-master-kubelet                2c9371fbb673b97a6fe8b1c52…     3.2.0            5h18m
-...
-rendered-master-dde...           2c9371fbb673b97a6fe8b1c52...   3.2.0            5h18m
-rendered-worker-fde...           2c9371fbb673b97a6fe8b1c52...   3.2.0            5h18m
+NAME      CONFIG                    UPDATED  UPDATING   DEGRADED  MACHINECOUNT  READYMACHINECOUNT  UPDATEDMACHINECOUNT DEGRADEDMACHINECOUNT  AGE
+master    rendered-master-06c9c4…   True     False      False     3             3                  3                   0                     4h42m
+worker    rendered-worker-f4b64…    False    True       False     3             2                  2                   0                     4h42m
 ----
 +
-Note that the `machineconfigs` listed as `rendered` are not meant to be changed or deleted. Expect them to be hidden at some point in the future.
+where:
++
+--
+** `UPDATED`. The `True` status indicates that the MCO has applied the current machine config, as specified in the `STATUS` field in the `oc get mcp` output, to the nodes in that MCP. The `False` status indicates a node in the MCP is updating.
```

Comment:
You could state the full name before the first use of the "MCP" acronym in this module:

> machine config pool (MCP)